explorer/app/viewModelBuilders/anvil-catalog/common/viewModelBuilders.ts

```diff
@@ -130,6 +137,20 @@ export const buildParticipantCount = (
   };
 };
 
+/**
+ * Build props for Description component from the given entity response.
+ * TODO revisit - separate from entity builder, generalize description component, revisit transformer
+ * @param response - Response model return from datasets API.
+ * @returns model to be used as props for the Description component.
+ */
+export const buildStudytDescription = (
```

Comment:
Refactor method name from `buildStudytDescription` to `buildStudyDescription`.

Reply:
SB Fixed.